application/src/main/kotlin/net/corda/v5/application/services/TokenClaimCriteria.kt

```diff
@@ -0,0 +1,28 @@
+package net.corda.v5.application.services
```

Comment:
Is `application` the correct place to put this code?

Is it a `ledger` concern instead? Should it be in the `ledger-utxo` module?

Reply:
Interesting, I see token selection (and in the future the other corda services) as platform services for the flows to use, although the token/state data comes from the ledger I'm not sure the service lives there, but I don't have a strong opinion, do we have other flow facing injectable APIs that don't live in application?

Reply:
Yes, for example we're adding new services in the ledger code to be consumed by flows.

Reply:
Moved these and the avro types to be under the ledger.utxo namespace